components/cd-disclosure/cd-disclosure.css

```diff
@@ -0,0 +1,12 @@
+/**
+ * CD Disclosure
+ *
+ * This component allows you to implement a show/hide button for content while
+ * retaining a portion of it that is always visible. Typical use-cases are to
+ * show a "teaser" but allow the user to expand a longer-form piece of text.
+ *
+ * @see https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/
+ */
+.cd-disclosure {
+  display: none;
```

Comment:
Not sure about this simple default because that means we need to set the visibility explicitly (the rules with `display:block` in cd-disclosure-states) which may be problematic for the styling of the content because it means an implementor would have to override the rule if they want `display: flex;` for example instead of the `display: block`.

Instead I think we can have the expected behavior with only 1 rule and 1 class (assuming the removal of the `data-cd-disclosure-show-by-default` and the addition of the class as mentioned in the other comment below):

```css
.cd-disclosure:not([data-cd-disclosure-show-by-default="true"], .cd-disclosure--is-expanded) {
  display: none;
}
```

Related to that is hiding the content the best default when javascript is disabled? If not, we could a `.js ` before the rule(s).

Reply:
Really good point about the `.js` class. I'll take the above suggestions and consolidate into one file.